percy/lib/cli_wrapper.py

```diff
@@ -31,7 +31,7 @@ def is_percy_enabled():
                 log(f'Unsupported Percy CLI version, {version}')
                 return False
 
-            return True
+            return data['type']
```

Comment:
is_percy_enabled() should ideally true/false only depending on status. you can set another static var similar to `Environment.percy_build_id`  for type and use that for your use case.